### DISCUSSION TITLE
Latest mtail image i.e. 3.0.1 is missing mtail binary

### DISCUSSION THREAD

**Discussion by ns-ravinderk** [google/mtail#880] [Q&A] [answered]
Latest `mtail` image is not working. It fails with below error.
```
docker run -it ghcr.io/google/mtail:3.0.1 --help
docker: Error response from daemon: failed to create task for container: failed to create shim task: OCI runtime create failed: runc create failed: unable to start container process: exec: "mtail": executable file not found in $PATH: unknown. 
```
Also, the old tags seems to be removed. Can someone help please ?

**Comment by ns-ravinderk** [comment]
@jaqx0r Could you please have a look at it.

**Comment by jaqx0r** [comment]
The 3.0.1 container images are not good, and I can't figure out how to delete them all.

You should use 3.0.0, and wait for 3.0.2 to get released.

**Comment by jaqx0r** [comment]
Huh, there is no 3.0.0?!

I will make a new 3.0.2 release now.  I don't know what is going on with the image though, if there's no binary in it.

**Comment by jaqx0r** [comment]
https://github.com/google/mtail/releases/tag/v3.0.2 is released.

https://github.com/google/mtail/pkgs/container/mtail/229019588?tag=3.0.2

**Reply by ns-ravinderk** [reply]
Thank you @jaqx0r for looking into it quickly. 
Although, v3.0.2 is also failing with no binary.

```
docker run -it ghcr.io/google/mtail:3.0.2 --help
Status: Downloaded newer image for ghcr.io/google/mtail:3.0.2
docker: Error response from daemon: failed to create task for container: failed to create shim task: OCI runtime create failed: runc create failed: unable to start container process: exec: "mtail": executable file not found in $PATH: unknown.
```

**Reply by jaqx0r** [reply]
Ok, thanks!

The release must be building the image incorrectly.

**Reply by jaqx0r** [reply, marked as answer]
I think this is a bug in the Dockerfile, actually!

Fix pending.

**Reply by jaqx0r** [reply]
Here, try 3.0.3

**Reply by ns-ravinderk** [reply]
Thank you @jaqx0r . `3.0.3` seems to be working fine.
